packages/artillery/lib/cmds/run.js

```diff
@@ -529,6 +536,20 @@ async function sendTelemetry(script, flags, extraProps) {
   }
 }
 
+function checkDirExists(output) {
+  if (!output) {
+    return;
+  }
+  // If destination is a file check only path to its directory
+  const exists = path.extname(output)
+    ? fs.existsSync(path.dirname(output))
+    : fs.existsSync(output);
+
+  if (!exists) {
+    throw new Error('No such directory!');
```

Comment:
Would be nice to catch this error and print a user-friendly message, e.g. `Directory x/y/z does not exist`.